### PR TITLE
List milkTIA asset 

### PR DESCRIPTION
**Pull request by josietyleung**
## Description

Adding token: milkTIA from MilkyWay protocol. milkTIA is the liquid staked token of TIA 

### Adding Assets

If adding a new asset, please ensure the following:
- [X] Asset is registered to the [Cosmos Chain Registry] https://github.com/cosmos/chain-registry/pull/3361
- [X] Add asset to bottom of `zone_assets.json`.

### On-chain liquidity

If adding a new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [X] The submitting team will ensure a pool will be created soon. Please hold off until further communication, which will notify of the Pool ID. (Please set the PR's status to Draft until the pool has been created)

OR
- [ ] A preview link is requested for the submitting team to be able to create a pool using the Osmosis Frontend.

OR
- [ ] The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds.

### CODE REVIEW

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -12302,6 +12302,30 @@
         "osmosis-main",
         "osmosis-price:ibc/498A0751C798A0D9A389AA3691123DADA57DAA4FE165D5C75894505B876BA6E4:1315"
       ]
+    },
+    {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:12306]: Please do not edit this file directly

**Comment by JeremyParish69** [comment]
Can we create a circular version of the token logo and add that as an image at the chain registry?

File: osmosis-1/osmosis-1.assetlist.json (hunk 1 above)
Comment (josietyleung) [reply osmosis-1/osmosis-1.assetlist.json]: removed edit of the file

**Comment by josietyleung** [comment]
> Can we create a circular version of the token logo and add that as an image at the chain registry?

Done!

**Comment by josietyleung** [comment]
> The validation check is failing because in the Chain Registry, the base denom is .../milkTIA, (capitalizaed TIA) while here it's defined as .../milktia (lowercase tia), and the capitalization does matter and must be consistent. Please confirm the correct denom and make the necessary correction so the validation check can pass.

hi updated to milkTIA to be consistent thanks!

**Pull request merged**